backend/jsonlogic/filter_test.go

```diff
@@ -0,0 +1,54 @@
+package jsonlogic
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestConvertToSQL(t *testing.T) {
+	// Open our jsonFile
+	jsonFile, err := os.Open("./testdata/filter.json")
```

Comment:
If you modify the test files a touch and make them something like:

```json
{
    "in": {}, // <- Json logic object
    "out": "X <> 99" // <- Expected result
}
```
then you can just run through all json files in that dir and we can super easily add new cases.